libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -275,18 +275,30 @@ HAL_Linux::HAL_Linux() :
 
 void _usage(void)
 {
-    printf("Usage: -A uartAPath -B uartBPath -C uartCPath -D uartDPath -E uartEPath -F uartFPath -G uartGpath -H uartHpath -I uartIpath -J uartJpath\n");
+    printf("Usage: --serial0 serial0Path --serial1 serial2Path \n");
     printf("Options:\n");
```

Comment:
I'd call this "Examples"

Reply:
Pre-existing issue; I'll create a follow-up PR.